### PR TITLE
🧵 Implement the Proposal Discussions thread

### PR DESCRIPTION
**Pull request by thesan**
Just the thread part no editor

See #868 
See #801

### CODE REVIEW

**Comment by thesan** [comment]
I realized there is still some work on "Allowing to mention someone" and "Post repay to selected message" on the editor so I will create another PR for that.

**Review by jodator — CHANGES_REQUESTED** [review]
Hey, overall nice progress with this :)

What's left:
- discussion mode [closed](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=2059%3A20)
  - closed should be grey
  - There should be info
- typo in the [reply to badge](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=2022%3A107529), should be: 'replied to ...'

File: packages/ui/src/proposals/components/ProposalDiscussions.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const ProposalDiscussionsStyles = styled.div<Pick<ProposalDiscussionThread, 'mod
           `
         : css`
             background-color: ${Colors.Red[50]};
-            color: ${Colors.Red[200]};
+            color: ${Colors.Black[75]};
```
Comment (jodator) [inline packages/ui/src/proposals/components/ProposalDiscussions.tsx]: background as well ;)

**Review by jodator — CHANGES_REQUESTED** [review]
Almost 🙃

**Comment by jodator** [comment]
@thesan The thread works nice :rocket: I've fixed the badge and I'm going to marge it after the CI.

**Pull request merged**